### DISCUSSION TITLE
Display Angular's directives in HTML component

### DISCUSSION THREAD

**Discussion by ghost** [LunarVim/LunarVim#2073] [Q&A] [answered]
Hello! I was wondering if it's a lack of configuration in my lunarvim config file or that [nvim-lsp-installer](https://github.com/williamboman/nvim-lsp-installer) doesn't support it, but I can't see auto completion for Angular's directives in HTML component files, like the one [this coc angular plugin](https://github.com/iamcco/coc-angular) supports, can anyone help me?

This is what I want (taken from coc-angular plugin):
![grafik](https://user-images.githubusercontent.com/23178554/145840368-dde9fabf-9a56-4cf4-a2f0-1a9c43412261.png)

This is what I have:
![grafik](https://user-images.githubusercontent.com/23178554/145839968-fe29cfc0-dc95-444c-b075-5772bfacf008.png)

My LSP Info:
![grafik](https://user-images.githubusercontent.com/23178554/145840188-de6d6295-9180-44ee-a721-0e9eacb71a44.png)

**Comment by ghost** [comment]
Hello! I'm trying again, to setup angularls with this config:
```lua
vim.list_extend(lvim.lsp.override, { "angularls" })
local opts = {
  cmd = {"~/.local/share/nvim/lsp_servers/angularls/node_modules/@angular/language-serverngserver/bin/ngserver", "--stdio", "--tsProbeLocations", project_library_path , "--ngProbeLocations", project_library_path}
}
require("lvim.lsp.manager").setup("angularls", opts)
```

but then I get: "Spawning language server with cmd: `ngserver` failed. The language server is either not installed, missing from PATH, or not executable.", while opening an HTML file.

![grafik](https://user-images.githubusercontent.com/23178554/149579881-db0201cb-daa7-480b-bbad-97d3e458ab6c.png)

**Comment by Doomwhite** [comment]
Found any success?

**Reply by ghost** [reply]
This is not a good answer in my opinion, because it messes with the core/default settings, that being said, commenting out `"angularls"` from `"override"` list (in "~/.local/share/lunarvim/lvim/lua/lvim/lsp/config.lua") does the job.

**Comment by LostInTheLogs** [comment, marked as answer]
```lua
lvim.lsp.automatic_configuration.skipped_servers = vim.tbl_filter(function(server)
  return server ~= "angularls"
end, lvim.lsp.automatic_configuration.skipped_servers)
```
restart lvim and run :LvimCacheReset
